src/main/java/org/folio/linked/data/mapper/dto/monograph/common/CategorySetMapperUnit.java

```diff
@@ -0,0 +1,45 @@
+package org.folio.linked.data.mapper.dto.monograph.common;
+
+import static org.folio.ld.dictionary.PropertyDictionary.LABEL;
+import static org.folio.ld.dictionary.PropertyDictionary.LINK;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.CATEGORY;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.CATEGORY_SET;
+import static org.folio.linked.data.util.BibframeUtils.putProperty;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+import org.folio.linked.data.mapper.dto.common.CoreMapper;
+import org.folio.linked.data.model.entity.Resource;
+import org.folio.linked.data.service.HashService;
+
+public abstract class CategorySetMapperUnit extends CategoryMapperUnit {
+
+  private final CoreMapper coreMapper;
+  private final HashService hashService;
+
+  protected CategorySetMapperUnit(CoreMapper coreMapper, HashService hashService) {
+    super(coreMapper, hashService, CATEGORY);
```

Comment:
We can hardcode `private final ResourceTypeDictionary type = CATEGORY` in `CategoryMapperUnit`. This way, we can avoid passing `CATEGORY` as the third param in the `super` call.

Reply:
updated with changes